data-prepper-plugins/geoip-processor/README.md

```diff
@@ -46,9 +46,9 @@ pipeline:
 
 - `keys` (Required) : List of properties like source, target and attributes can be specified where the location fields are written
 
-  - `source` (Required) : source IP for which enrichment will be done. Public IP can be either IPV4 or IPV6.
+  - `source` (Required) : source  and destination IP's for which enrichment will be done. Public IP's can be either IPV4 or IPV6.
```

Comment:
Im a little confused, what does destination IP refer to? Does this just mean if there are two IPs that want to be looked up?

Reply:
Yes. There can be multiple IP's with  "," is the delimiter with any name. However the same name should be present in YAML cinfiguration for parsing and reading the IP. For example as below:
source: ["/peer/ips/src_ip", "/peer/ips/dest_ip"]